test-db-run-repl.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
Remove the whole file, it is just an ad-hoc test script?

Reply:
yes

Reply:
its for easier invoking test command